templates/alert_form.html.twig

```diff
@@ -0,0 +1,252 @@
+    {% set base_option = {
+        'is_horizontal' : false,
+    } %}
+    {% set base_option_small = {
+        'field_class' : 'col-xxl-4 col-xl-6',
+    }|merge(base_option) %}
+
+    {{ fields.textField(
+        'name',
+        item.fields.name,
+        __('Name'),
+        base_option
+    ) }}
+    {{ fields.dropdownYesNo(
+        'is_active',
+        item.fields.is_active,
+        __('Active'),
+        base_option
+    ) }}
+
+    {{ fields.textareaField(
+        'message',
+        item.fields.message,
+        __('Description'),
+        {
+            'full_width': true,
+            'enable_richtext': true,
+            'enable_images'  : false,
+        }|merge(base_option)
+    ) }}
+
+    {{ fields.dateTimeField(
+        'date_start',
+        item.fields.date_start,
+        __('Visibility start date'),
+        base_option
+    ) }}
+     {{ fields.dateTimeField(
+        'date_end',
+        item.fields.date_end,
+        __('Visibility end date'),
+        base_option
+    ) }}
+
+    {{ news_fields.checkboxesField(
+        "show_on_pages",
+        {
+            'is_displayed_oncentral' : __('Central page', 'news'),
+            'is_displayed_onlogin'   : __('Login page', 'news'),
+            'is_displayed_onhelpdesk': __('Helpdesk page', 'news'),
+        },
+        item.fields,
+        __('Show on pages', 'news'),
+        base_option
+    )}}
+    {{ fields.dropdownYesNo(
+        'is_close_allowed',
+        item.fields.is_close_allowed,
+        __("Can close alert", 'news'),
+        base_option
+    ) }}
+
+    <div class="hr-text">
+        <i class="ti ti-palette"></i>
+        <span>{{ __('Customization', 'news') }}</span>
+    </div>
+
+    {{ fields.dropdownArrayField(
+        'type',
+        item.fields.type,
+        templates,
+        __("Template"),
+        {
+            'display_emptychoice': true,
+        }|merge(base_option_small)
+    ) }}
+    {{ fields.nullField() }}
+
+    {{ fields.dropdownArrayField(
+        'size',
+        item.fields.size,
+        sizes,
+        __("Size"),
+        base_option_small
+    ) }}
+    {{ news_fields.iconField(
+        "icon",
+        icons,
+        item.fields.icon,
+        __('Icon'),
+        {
+            'display_emptychoice': true,
+        }|merge(base_option_small)
+    ) }}
+    {{ fields.dropdownYesNo(
+        'display_dates',
+        item.fields.display_dates,
+        __('Show dates'),
+        base_option_small
+    ) }}
+
+    {{ news_fields.colorfield(
+        "background_color",
+        colors,
+        item.fields.background_color,
+        __('Background color'),
+        base_option_small
+    )}}
+    {{ news_fields.colorfield(
+        "text_color",
+        colors,
+        item.fields.text_color,
+        __('Text color'),
+        base_option_small
+    )}}
+    {{ news_fields.colorfield(
+        "accent_color",
+        colors,
+        item.fields.accent_color,
+        __('Accent color'),
```

Comment:
@cconard96 
Should we use `Accent color` or `Emphasis color`, or something else here?

Reply:
ping @cconard96